examples/TypeScriptMessaging/App.tsx

```diff
@@ -0,0 +1,402 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { SafeAreaView, TouchableOpacity, View } from 'react-native';
+import { NavigationContainer, RouteProp } from '@react-navigation/native';
+import {
+  createStackNavigator,
+  StackNavigationProp,
+} from '@react-navigation/stack';
+import { enableScreens } from 'react-native-screens';
+import { ChannelSort, Channel as ChannelType, StreamChat } from 'stream-chat';
+import {
+  Channel,
+  ChannelList,
+  Chat,
+  CloseButton,
+  MessageInput,
+  MessageList,
+  Streami18n,
+  Thread,
+  ThreadContextValue,
+} from 'stream-chat-react-native';
+
+enableScreens();
+
+type LocalAttachmentType = {};
+type LocalChannelType = { dummyProp: string };
+type LocalCommandType = '';
+type LocalEventType = {};
+type LocalMessageType = {};
+type LocalResponseType = {};
+type LocalUserType = {};
+
+// Read more about style customizations at - https://getstream.io/chat/react-native-chat/tutorial/#custom-styles
+const theme = {
+  avatar: {
+    image: {
+      css: `
```

Comment:
we going template literal vs object on these?

Reply:
just showing how it works right now, we can do whatever. might customize it more with the themed changes Neil is making. this is also a super simple example implementation instead of an in depth showcase